src/envo/envo-edit.owl

```diff
@@ -21891,17 +21891,18 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/ecozone") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000280> "OBSOLETE. Ecozones delineate large areas of a planetary surface within which organisms have been evolving in relative isolation over long periods of time, separated from one another by geographic features, such as oceans, broad deserts, or high mountain ranges, that constitute barriers to migration.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0100001> <http://purl.obolibrary.org/obo/ENVO_01000280> <http://purl.obolibrary.org/obo/ENVO_03620000>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_01000280> "ORCID:0000-0002-4366-3088")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_01000280> "2013-10-13T18:32:46Z")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01000280> <http://purl.obolibrary.org/obo/envo#envoPolar>)
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01000280> "ENVO contains this top-level class, but all instances will be in GAZ. The definition is preliminary and will be aligned to BFO.
 https://en.wikipedia.org/wiki/  Ecozones correspond to the floristic kingdoms of botany or zoogeographic regions of zoology. Ecozones are characterized by the evolutionary history of the organisms they contain. They are distinct from biomes, also known as major habitat types, which are divisions of the Earth's surface based on life form, or the adaptation of plants and animals to climatic, soil, and other conditions. Biomes are characterized by similar climax vegetation. Each ecozone may include a number of different biomes. A tropical moist broadleaf forest in Central America, for example, may be similar to one in New Guinea in its vegetation type and structure, climate, soils, etc., but these forests are inhabited by plants and animals with very different evolutionary histories.")
-AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01000280> "ecozone")
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000280> <http://purl.obolibrary.org/obo/ENVO_01001110>)
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000280> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_01000817>))
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01000280> "term replaced by 'biogeographic realm' ENVO:03620000")
```

Comment:
Use 'term replaced by'

See

- https://oboacademy.github.io/obook/howto/obsolete-term/

Reply:
That's the tutorial I followed and I can see the annotation on the branch in Protege. Entirely possible I messed something up but I'm not sure exactly what or how to fix it.

![Screenshot 2025-02-03 153258](https://github.com/user-attachments/assets/e84b962e-203f-498b-8912-f7aed6e85b18)

Reply:
The comment is redundant.

```suggestion
```

The correct way is what you did above:

https://github.com/EnvironmentOntology/envo/pull/1582/files#diff-9a5b50d513c6b7f58aeb2d37743df7b38d8552400cb9daed1ff48a7f0b7e2b37R21897

Reply:
Resolved by latest commit. Should be ready to merge.